docs/BUG-BOUNTY.md

```diff
@@ -0,0 +1,86 @@
+# The curl bug bounty
+
+The curl project runs a bug bounty program in association with
+[HackerOne](https://www.hackerone.com/).
+
+# How does it work?
+
+Start out by posting your suspected security vulnerability directly to [curl's
+hackerone security bug tracker](https://www.hackerone.com/curl).
+
+After you have reported a security issue, it has been deemed credible and a
+patch and advisory has been made public you can be eligible for a bounty from
+this program.
+
+See all details at https://hackerone.com/curl
```

Comment:
Should this be a Markdown link like `[https://hackerone.com/curl](https://hackerone.com/curl)`?

Reply:
Fixed!